README.md

```diff
@@ -1,3 +1,4 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15564919&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
```

Comment:
1. software engineering is define as the process of using scientific principle to craete software,it uses the systematic approach to design,build and test software solution.
2.  Identify and describe at least three key milestones in the evolution of software engineering.
1. Structured Programming (1960s-1970s): Brought in clear control structures, like loops and conditionals, making code easier to read and less error-prone.
2. Object-Oriented Programming (1980s-1990s): Focused on organizing code around objects, promoting modularity, reuse, and easier maintenance.
3. Agile Methodologies (2000s-present): Changed how teams work by emphasizing flexibility, collaboration, and rapid, iterative development to quickly deliver high-quality software.